docs/concepts.md

```diff
@@ -1,15 +1,98 @@
-# TriggerMesh concepts
+The following diagram depicts the core concepts of the TriggerMesh Cloud-Native Integration Platform: Source, Target, Function, Transformation, Filter, Splitter.
 
-## Source
+![](./assets/images/concepts.png)
+
+TriggerMesh allows you to integrate applications (on-premises or cloud services) by defining event sources and targets which are seen as the start and the end of a so-called Bridge. While events flow through the Bridge they can undergo splitting, filtering and transformation. Transformation of events can be done declaratively or through a function.
```

Comment:
Playing the role of a reader who's unfamiliar with TriggerMesh:

_OK Transformations can be done declaratively or through a function. But the text also mentions "Splitting" and "Filtering". Can I do these either declaratively or via a function too?_

I think it's a fair question to answer since the last sentence sounds over-focused on one particular aspect and onits the others completely.